<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>417</width>
     <height>626</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_toMenu">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(201, 160, 220);
}
</string>
   </property>
   <property name="text">
    <string>Вернуться в меню</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
